src/common/requires/removeUnusedTypes.js

```diff
@@ -43,7 +43,7 @@ const CONFIG: Array<ConfigEntry> = [
 
 function removeUnusedTypes(root: Collection, options: SourceOptions): void {
   const declared = getDeclaredIdentifiers(root, options);
-  const used = getNonDeclarationTypes(root, options);
+  const used = getNonDeclarationTypes(root);
```

Comment:
This fixes a Flow error that was surfaced when I upgraded recently (it was always there).